mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      acknowledge SOSE data in all products or publications that use them, e.g.
+      by including the following written note: "Computational resources for the
+      SOSE were provided by NSF XSEDE resource grant OCE130007." An appropriate
+      citation should also be made.
+    </releasePolicy>
+    <references>
+      [Mazloff et al. (2010)](http://doi.org/10.1175/2009JPO4236.1)
+    </references>
+    <bibtex>
+      @article{Mazloff2010,
+      author = {Matthew R.  Mazloff  and  Patrick  Heimbach  and  Carl  Wunsch},
+      title = {An Eddy-Permitting Southern Ocean State Estimate},
+      journal = {Journal of Physical Oceanography},
+      volume = {40},
+      number = {5},
+      pages = {880-899},
+      year = {2010},
+      doi = {10.1175/2009JPO4236.1},
+      URL = {https://doi.org/10.1175/2009JPO4236.1}
+      }
+    </bibtex>
+    <dataUrls>
+      - http://sose.ucsd.edu/DATA/SO6_V2/THETA_mnthlyBar.0000000100.data.gz
+      - http://sose.ucsd.edu/DATA/SO6_V2/THETA_mnthlyBar.0000000100.meta
+      - http://sose.ucsd.edu/DATA/SO6_V2/SALT_mnthlyBar.0000000100.data.gz
+      - http://sose.ucsd.edu/DATA/SO6_V2/SALT_mnthlyBar.0000000100.meta
+    </dataUrls>
+    <preprocessing>
+      preprocess_observations/remap_SOSE_T_S.py
+    </preprocessing>
+    <tasks>
+      - climatologyMapSoseTemperature
+      - climatologyMapSoseSalinity
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      Antarctic melt rates and fluxes
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      Melt rates and melt fluxes from Rignot et al. (2013)
+    </description>
+    <source>
+      [Ice-Shelf Melting Around Antarctica](http://science.sciencemag.org/content/341/6143/266)
+    </source>
+    <releasePolicy>
+      Data available upon request from co-author J. Mouginot.
+    </releasePolicy>
+    <references>
+      [Rignot et al. (2013)](http://science.sciencemag.org/content/341/6143/266)
+    </references>
+    <bibtex>
+      @article{Rignot2013,
+	  title = {Ice-{Shelf} {Melting} {Around} {Antarctica}},
+	  volume = {341},
+	  url = {http://www.ncbi.nlm.nih.gov/pubmed/23765278},
+	  doi = {10.1126/science.1235798},
+	  number = {6143},
+	  journal = {Science},
+	  author = {Rignot, E. and Jacobs, S. and Mouginot, J. and Scheuchl, B.},
+	  month = jul,
+	  year = {2013},
+	  pages = {266--270}
+	  }
+    </bibtex>
+    <dataUrls>
+      - http://science.sciencemag.org/highwire/filestream/594977/field_highwire_adjunct_files/0/1235798tableS1.xlsx
+      - (data available upon request from J. Mouginot)
+    </dataUrls>
+    <preprocessing>
+      preprocess_observations/remap_rignot.py
+    </preprocessing>
+    <tasks>
+      - climatologyMapAntarcticMelt
+      - timeSeriesAntarcticMelt
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      HadISST Nino 3.4 Index
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      (missing)
+    </description>
```

Comment:
@vanroekel, can you help fill in the missing info about the 2 Nino 3.4 data sets?